example/rater/classification.ipynb

```diff
@@ -1,8 +1,31 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use AutoRater to classify answer label from a Jupyter Notebook\n",
+    "\n",
+    "In this example, we will show you how to use autorater to classify answer label from a given jupyter notebook.\n",
```

Comment:
"In this example, we will show you how to use `AutoRater` to verify the correctness of an answer to a given question and context pairs."